core/src/main/java/org/arquillian/smart/testing/Configuration.java

```diff
@@ -45,6 +45,10 @@ public boolean isModeSet() {
         return this.mode != null;
     }
 
+    public boolean isSkipTests() {
+        return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
```

Comment:
The thing to check here would be if `-DskipTests` does not imply `maven.test.skip` set implictly.